cmd/zpool/Makefile.am

```diff
@@ -2,7 +2,8 @@ include $(top_srcdir)/config/Rules.am
 
 DEFAULT_INCLUDES += \
 	-I$(top_srcdir)/include \
-	-I$(top_srcdir)/lib/libspl/include
+	-I$(top_srcdir)/lib/libspl/include \
+	-I.
```

Comment:
Why was this needed?  `zpool_util.h` hasn't moved.

Reply:
It's relative location has changed.